.prow.yaml

```diff
@@ -0,0 +1,14 @@
+presubmits:
+  GoogleCloudPlatform/blueprints:
+    - name: test
+      decorate: true
```

Comment:
add `cluster: build-blueprints`

Reply:
Do we need this explicitly specified for every job?

Reply:
yes